src/plugins/data/public/search/aggs/buckets/date_histogram.ts

```diff
@@ -202,7 +198,8 @@ export const getDateHistogramBucketAgg = ({
             ...dateHistogramInterval(interval.expression),
           };
 
-          const scaleMetrics = scaleMetricValues && interval.scaled && interval.scale < 1;
+          const scaleMetrics =
+            scaleMetricValues && interval.scaled && interval.scale && interval.scale < 1;
```

Comment:
nit: not sure we need to check for `interval.scale`, if the value is undefined `interval.scale < 1 `also returns false

Reply:
Typescript thinks that we should(